<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1118</width>
    <height>681</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Eras Medium ITC</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);
background-color: rgb(255, 255, 255);</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="nome_produto">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>220</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Eras Medium ITC</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>NOME DO PRODUTO</string>
    </property>
   </widget>
   <widget class="QLabel" name="tamanho">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>430</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Eras Medium ITC</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TAMANHO</string>
    </property>
   </widget>
   <widget class="QLabel" name="modelo">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>320</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Eras Medium ITC</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>MODELO</string>
    </property>
   </widget>
   <widget class="QLabel" name="cor">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>320</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Eras Medium ITC</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>COR</string>
    </property>
   </widget>
   <widget class="QLabel" name="marca">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>320</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Eras Medium ITC</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>MARCA</string>
    </property>
   </widget>
   <widget class="QLabel" name="codigo">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>220</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Eras Medium ITC</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>CODIGO</string>
    </property>
   </widget>
   <widget class="QLabel" name="qntdd_recebida">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>430</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Eras Medium ITC</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>QUANTIDADE RECEBIDA</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inserir_nome_produto">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>270</y>
      <width>381</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="inserir_modelo">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>370</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="inserir_marca">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>370</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="inserir_qntd_recebida">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>480</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="inserir_cor">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>370</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="inserir_cod">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>270</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="preco_venda">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>220</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Eras Medium ITC</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>PREÇO VENDA</string>
    </property>
   </widget>
   <widget class="QLabel" name="preco_custo">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>220</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Eras Medium ITC</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>PREÇO CUSTO</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inserir_preco_custo">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>270</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="inserir_preco_venda">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>270</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-11</y>
      <width>1121</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(161, 193, 190);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="titulo">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>30</y>
       <width>631</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Eras Demi ITC</family>
       <pointsize>36</pointsize>
      </font>
     </property>
     <property name="text">
      <string>CADASTRO DO PRODUTO</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="logo">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>-50</y>
       <width>231</width>
       <height>251</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>Meteoro_Calçados_Logo-removebg-preview.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>1121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(36, 36, 36);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_cadastrar">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>550</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Eras Demi ITC</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(36, 36, 36);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>CADASTRAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_ver_estoque">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>160</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Eras Demi ITC</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(36, 36, 36);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>VER ESTOQUE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_apagar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>550</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(36, 36, 36);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>image.png</normaloff>image.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>55</width>
      <height>55</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_voltarcadprod">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(36, 36, 36);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>pngtree-arrow-left-flat-white-color-icon-move-back-less-photo-picture-image_7920547-removebg-preview.png</normaloff>pngtree-arrow-left-flat-white-color-icon-move-back-less-photo-picture-image_7920547-removebg-preview.png</iconset>
    </property>
   </widget>
   <widget class="QLineEdit" name="inserir_tamanho">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>480</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_atualizar">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>550</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Eras Demi ITC</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(36, 36, 36);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>ATUALIZAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_consultar">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>550</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Eras Demi ITC</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(36, 36, 36);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>CONSULTAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_deletar">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>550</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Eras Demi ITC</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(36, 36, 36);</string>
    </property>
    <property name="text">
     <string>EXCLUIR</string>
    </property>
   </widget>
   <zorder>frame</zorder>
   <zorder>nome_produto</zorder>
   <zorder>tamanho</zorder>
   <zorder>modelo</zorder>
   <zorder>cor</zorder>
   <zorder>marca</zorder>
   <zorder>codigo</zorder>
   <zorder>qntdd_recebida</zorder>
   <zorder>inserir_nome_produto</zorder>
   <zorder>inserir_modelo</zorder>
   <zorder>inserir_marca</zorder>
   <zorder>inserir_qntd_recebida</zorder>
   <zorder>inserir_cor</zorder>
   <zorder>inserir_cod</zorder>
   <zorder>preco_venda</zorder>
   <zorder>preco_custo</zorder>
   <zorder>inserir_preco_custo</zorder>
   <zorder>inserir_preco_venda</zorder>
   <zorder>frame_2</zorder>
   <zorder>bt_cadastrar</zorder>
   <zorder>bt_ver_estoque</zorder>
   <zorder>bt_apagar</zorder>
   <zorder>bt_voltarcadprod</zorder>
   <zorder>inserir_tamanho</zorder>
   <zorder>bt_atualizar</zorder>
   <zorder>bt_consultar</zorder>
   <zorder>bt_deletar</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1118</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>inserir_cod</tabstop>
  <tabstop>inserir_nome_produto</tabstop>
  <tabstop>inserir_preco_custo</tabstop>
  <tabstop>inserir_preco_venda</tabstop>
  <tabstop>inserir_modelo</tabstop>
  <tabstop>inserir_cor</tabstop>
  <tabstop>inserir_marca</tabstop>
  <tabstop>inserir_qntd_recebida</tabstop>
  <tabstop>inserir_tamanho</tabstop>
  <tabstop>bt_cadastrar</tabstop>
  <tabstop>bt_ver_estoque</tabstop>
  <tabstop>bt_apagar</tabstop>
  <tabstop>bt_voltarcadprod</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
